### DISCUSSION TITLE
🐞 Bug in backend crud user test.

### DISCUSSION THREAD

**Discussion by dmenisdev** [fastapi/full-stack-fastapi-template#1407] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
def test_check_if_user_is_active_inactive(db: Session) -> None:
    email = random_email()
    password = random_lower_string()
    user_in = UserCreate(email=email, password=password, disabled=True)
    user = crud.create_user(session=db, user_create=user_in)
    assert user.is_active

There is no "disable" field in Class UserCreate, it should "is_active".

Fixed code:
def test_check_if_user_is_active_inactive(db: Session) -> None:
    email = random_email()
    password = random_lower_string()
    user_in = UserCreate(email=email, password=password, is_active=True)
    user = crud.create_user(session=db, user_create=user_in)
    assert user.is_active
```

### Description

Testing backend with: `bash ./scripts/test.sh` will not run correctly.

### Operating System

macOS

### Operating System Details

Apple silicon.

### Python Version

Python 3.12.4

### Additional Context

Came a across this bug and could only find `discussions` to report it.

**Comment by alejsdev** [comment]
Hello @dmenisdev, thank you for the report. This was already handled in #1479! :tada:
